tools/modules/EnsEMBL/Web/Component/Tools/VR/Results.pm

```diff
@@ -319,6 +360,110 @@ sub linkify {
   return @return_values;
 }
 
+sub linkify_mane {
+  my $self = shift;
+  my $field = shift;
+  my $values = shift;
+  my $species = shift;
+  my $job_data = shift;
+  my $mane_count_g = shift;
+
+  # work out core DB type
+  my $db_type = 'core';
+  if(my $ct = $job_data->{core_type}) {
+    if($ct eq 'refseq' || ($values && $ct eq 'merged' && $values !~ /^ENS/)) {
+      $db_type = 'otherfeatures';
+    }
+  }
+
+  my @return_values = ();
+  my %return_values_hgvsg;
+  my $new_value;
+  my $hub = $self->hub;
+  my $sd = $hub->species_defs;
+
+  my @all_values = split(', ', $values);
+
+  foreach my $value (@all_values) {
+    return '-' unless defined $value && $value ne '';
+
+    my $mane_result;
+    my ($hgvsg, $hgvsc, $hgvsp) = split /\;/, $value;
+
+    if($hgvsc =~ /^ENST/) {
```

Comment:
I'm 99% sure the answer to this question is no, but do we have any mane transcripts that don't begin with ENST?

Reply:
No, because MANE only applies to human and all human transcripts will have an ENST stable id. From the NCBI website: "The goal of this project is to provide a minimal set of matching RefSeq and Ensembl transcripts of human protein-coding genes"